CHANGES.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## Unreleased
+
+### _Black_
+
+- Fixed non-Python cell magics sometimes failing due to indentation (#2630)
```

Comment:
I think this can be made more explicit - can you make it clearer what the behaviour was before, and what it is now, so it's clear that `black` is now intentionally more timid about processing cell magics?

Reply:
addressed in https://github.com/psf/black/pull/2630/commits/1e8a0b1246e74406731981830a05203a35ad65bd